### PR TITLE
Task List Ordering Bug

### PR DESCRIPTION
**Pull request by josephkane**
## Type

- [x] Fix (a **bug** or other **issue** was fixed)
- [ ] Feature (**new functionality** was added)
- [ ] Breaking (**existing functionality** was affected)
- [ ] Other (please **explain** below)

## Details

* order incoming tasks via concat of type and id

### Additional Details?

* NOTE: this commit sorts tasks lexigraphically to squash the task jumping bug,
which means that the tasks aren't displayed in the order they occur. Will need some
coordination with the backend to order them properly

## Checklist

- [ ] [Documentation](https://supergiant.readme.io/v2.1.0-alpha/) was **needed**, and I updated it.
- [x] [Documentation](https://supergiant.readme.io/v2.1.0-alpha/) was **not** needed.
- [x] I understand [the Contribution Guidelines](https://supergiant.readme.io/docs/guidelines).

### CODE REVIEW

File: cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts [hunk 1]
```diff
@@ -191,14 +191,40 @@ export class ClusterComponent implements AfterViewInit, OnDestroy {
     a.click();
   }
 
+  orderTasks(a, b) {
```
Comment (taaraora) [inline cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts]: Can we plan to add models in the future?

File: cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts [hunk 2]
```diff
@@ -191,14 +191,40 @@ export class ClusterComponent implements AfterViewInit, OnDestroy {
...
+
   getKube() {
     // TODO: shameful how smart this ENTIRE component has become.
     this.subscriptions.add(observableTimer(0, 10000).pipe(
       switchMap(() => this.supergiant.Kubes.get(this.clusterId))).subscribe(
         k => {
           this.kube = k;
           // for dev-ing
-          // this.kube.state = "prepare";
+          this.kube.state = "prepare";
```
Comment (taaraora) [inline cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts]: oops

File: cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts (hunk 1 above)
Comment (josephkane) [reply cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts]: absolutely. for this whole feature (cluster details) honestly

File: cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts (hunk 2 above)
Comment (josephkane) [reply cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts]: thanks

**Pull request merged**